shared/login/recover-password/prompt-reset/container.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react'
+import * as Kb from '../../../common-adapters'
+import * as Styles from '../../../styles'
+import * as Container from '../../../util/container'
+import * as RecoverPasswordGen from '../../../actions/recover-password-gen'
+import DeviceSelector from '.'
+import {InfoIcon} from '../../../signup/common'
+
+type OwnProps = {}
+
+const ConnectedPromptReset = Container.connect(
+  _ => ({}),
+  dispatch => ({
+    onContinue: (action: boolean) => dispatch(RecoverPasswordGen.createSubmitResetPrompt({action})),
+  }),
+  (s, d, o: OwnProps) => ({...o, ...s, ...d})
+)(DeviceSelector)
+
+// @ts-ignore fix this
```

Comment:
please don't duplicate this ts-ignore. this is working elsewhere. just don't put it on the container, put it on the wrapped class

Reply:
base pr is oooold and it was my first larger fe change, im nearly done with getting this up to date